hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.3__index_init.sql

```diff
@@ -225,7 +225,7 @@ alter table if exists topic_message
     add constraint topic_message__pk primary key (consensus_timestamp, topic_id);
 create index if not exists topic_message__topic_id_timestamp
     on topic_message (topic_id, consensus_timestamp);
-create unique index if not exists topic_message__topic_id_seqnum
+create index if not exists topic_message__topic_id_seqnum
     on topic_message (topic_id, sequence_number, consensus_timestamp);
```

Comment:
remove `consensus_timestamp` from the indexed column
```suggestion
create index if not exists topic_message__topic_id_seqnum
    on topic_message (topic_id, sequence_number);
```